lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -59,24 +59,79 @@ def set_up_course(self, enable_subsection_grades=True):
         _ = anonymous_id_for_user(self.user, self.course.id)
         # pylint: enable=attribute-defined-outside-init,no-member
 
-    def test_score_changed_signal_queues_task(self):
+    @ddt.data(
+        ('lms.djangoapps.grades.tasks.recalculate_subsection_grade.apply_async', SCORE_CHANGED),
+        ('lms.djangoapps.grades.tasks.recalculate_course_grade.apply_async', COURSE_GRADE_UPDATE_REQUESTED)
+    )
+    @ddt.unpack
+    def test_signal_queues_task(self, enqueue_op, test_signal):
         """
-        Ensures that the SCORE_CHANGED signal enqueues a recalculate subsection grade task.
+        Ensures that the SCORE_CHANGED and COURSE_GRADE_UPDATE_REQUESTED signals enqueue the correct tasks.
         """
         self.set_up_course()
+        if test_signal == SCORE_CHANGED:
+            expected_args = self.score_changed_kwargs.values()
+        else:
+            expected_args = [self.score_changed_kwargs['user_id'], self.score_changed_kwargs['course_id']]
```

Comment:
Shouldn't the task be called with the same fields, regardless of which signal was fired?